pdns/pdnsutil.cc

```diff
@@ -2607,6 +2609,20 @@ try
 
   loadMainConfig(g_vm["config-dir"].as<string>());
 
+  if (cmds.at(0) == "lmdb-get-schema-version") {
+    // FIXME: add command to help
+    if(cmds.size() != 2) {
+      cerr << "Syntax: pdnsutil lmdb-get-schema-version /path/to/pdns.lmdb"<<endl;
+      return 0;
+    }
+
+    auto res = LMDBBackend::getSchemaVersionAndShards(cmds.at(1));
```

Comment:
I expected this command to just print the current schema version of this LMDB backend (constant value) instead of opening an LMDB and reading the schema version of that LMDB.

Reply:
there is `pdns/pdns_server --no-config --config --launch=lmdb | grep schema` but I'll add a separate command that emits a single number

Reply:
it would in fact be very convenient for me if I can drop the command that inspects an existing LMDB, would that be acceptable? The command that returns the supported version is easy.

Reply:
@nvaatstra

Reply:
Yep, no need for that command from my end